sparkmagic/sparkmagic/livyclientlib/endpoint.py

```diff
@@ -1,14 +1,15 @@
 from .exceptions import BadUserDataException
+import sparkmagic.utils.configuration as conf
 
 
 class Endpoint(object):
-    def __init__(self, url, username="", password=""):
+    def __init__(self, url, auth_type=conf.default_livy_endpoint_auth_type(), username="", password=""):
```

Comment:
I don't think we need a default value here, do we? If we don't need it, then we could get rid of the `default_livy_endpoint_auth_type` setting altogether.